kore/src/main/scala/org/kframework/compile/AddBottomSortForListsWithIdenticalLabels.scala

```diff
@@ -17,7 +17,7 @@ object AddBottomSortForListsWithIdenticalLabels extends Function[Module, Module]
         case (klabel, userListInfo) =>
           val minimalSorts = m.subsorts.minimal(userListInfo map { li => li.sort })
           if (minimalSorts.size > 1) {
-            val newBottomSort = Sort("GeneratedListBottom{" + klabel + "}")
+            val newBottomSort = Sort("GeneratedListBottom{" + klabel.name.replace("|","") + "}")
```

Comment:
What's going on here? Switching for `klabel.name()` makes sense to me, why remove the `|`?